test/unit_test/explore_test.cc

```diff
@@ -97,13 +97,13 @@ BOOST_AUTO_TEST_CASE(sample_continuous_action_statistical)
   float pdf_value;
   constexpr float range_min = .0f;
   constexpr float range_max = 100.0f;
-  uint64_t random_seed = 7791;
+  std::shared_ptr<rand_state> random_seed = std::make_shared<rand_state>(7791);
```

Comment:
```suggestion
  auto random_seed = std::make_shared<rand_state>(7791);
```

Reply:
I reverted explore_internal.h so seed can be a uint64_t here again